### PR TITLE
fix(event): Show workitemtype change event

### PR DESCRIPTION
**Pull request by jarifibrahim**
With this PR
 - Workitem type change event is shown in audit trail. 

The workitem type change event is shown as
screenshot from 2018-09-11 12-22-38

### CODE REVIEW

**Comment by jarifibrahim** [comment]
[test]

**Comment by sahil143** [comment]
[test]

**Comment by jarifibrahim** [comment]
>@jarifibrahim event logged is not correct.I updated the workitem type from 'Bug'
to 'Papercut' but it is logged Raunak
changed the workitem type from 'Papercuts' to 'Bug'

@Raunak1203 I've fixed it in https://github.com/fabric8-ui/fabric8-planner/pull/2783/commits/14485fbbaf0b58f23f345d28f712428a12d4e8ce . Please take a look at it again.

File: src/app/models/event.model.ts [hunk 1]
```diff
@@ -228,7 +230,19 @@ export class EventQuery {
...
+                  modifier: this.userQuery.getUserObservableById(event.modifierId),
+                  newValueRelationshipsObs: event.newValueRelationships.map(item => {
+                    return this.workitemTypeQuery.getWorkItemTypeById(item.id);
+                  }),
+                  oldValueRelationshipsObs: event.oldValueRelationships.map(item => {
+                    return this.workitemTypeQuery.getWorkItemTypeById(item.id);
+                  })
+                };
               default:
+                console.log('Unknown event: ', event);
```
Comment (sahil143) [inline src/app/models/event.model.ts]: Please comment here why this console.log is needed.

Comment (jarifibrahim) [reply src/app/models/event.model.ts]: What? This console log is self-explanatory. What else do you think should be added?

Comment (sahil143) [reply src/app/models/event.model.ts]: Just a comment saying don't remove this console and it logs the unknown event.

Comment (jarifibrahim) [reply src/app/models/event.model.ts]: Done.

**Pull request merged**